### ISSUE TITLE
Failed to build for Windows architecture

### ISSUE THREAD

**Issue by vania-pooh** [aandryashin/selenoid#7]
Some dependencies should be added to govendor:
```
vania-pooh:/selenoid$ GOOS=windows GOARCH=386 go build -o selenoid.exe
vendor/github.com/docker/go-connections/sockets/sockets_windows.go:7:2: cannot find package "github.com/Microsoft/go-winio" in any of:
	/src/testing/go/src/github.com/aandryashin/selenoid/vendor/github.com/Microsoft/go-winio (vendor tree)
	/usr/local/Cellar/go/1.7.1/libexec/src/github.com/Microsoft/go-winio (from $GOROOT)
	/src/testing/go/src/github.com/Microsoft/go-winio (from $GOPATH)
```
However:
```
$ go get github.com/Microsoft/go-winio
# github.com/Microsoft/go-winio
../../Microsoft/go-winio/file.go:45: undefined: syscall.Overlapped
```
This is when building under Mac.

**Comment by vania-pooh** [comment]
Same when building on Windows.

**Comment by vania-pooh** [comment]
```
vania-pooh:/src/testing/go/src/github.com/aandryashin/selenoid$ GOOS=windows GOARCH=386 go get github.com/Microsoft/go-winio
vania-pooh:/src/testing/go/src/github.com/aandryashin/selenoid$ GOOS=windows GOARCH=386 go build -o selenoid.exe
# github.com/aandryashin/selenoid/service
service/driver.go:44: unknown syscall.SysProcAttr field 'Setpgid' in struct literal
service/driver.go:66: undefined: syscall.Getpgid
service/driver.go:71: undefined: syscall.Kill
```

**Comment by aandryashin** [comment]
Actually drivers respond to close browsers on exit and they do it, so we don't need platform depended code to force that behaviour.

**Comment by aandryashin** [comment]
Fixed in ca5f57b7db4fe8d776b986118f72a009d0a4cdab.
